server/src/internalClusterTest/java/org/elasticsearch/readiness/ReadinessClusterIT.java

```diff
@@ -21,23 +21,28 @@
 import org.elasticsearch.core.Strings;
 import org.elasticsearch.core.Tuple;
 import org.elasticsearch.discovery.MasterNotDiscoveredException;
+import org.elasticsearch.plugins.Plugin;
 import org.elasticsearch.reservedstate.action.ReservedClusterSettingsAction;
 import org.elasticsearch.reservedstate.service.FileSettingsService;
 import org.elasticsearch.test.ESIntegTestCase;
 import org.elasticsearch.test.ESIntegTestCase.ClusterScope;
 import org.elasticsearch.test.InternalTestCluster;
-import org.elasticsearch.test.readiness.ReadinessClientProbe;
```

Comment:
ReadinessClientProbe is used in ReadinessServiceTests which opens ports too.
do we want to refactor it as well?

Reply:
Yes, I had the same question. I think it would be a good idea; do we want to make it now or as a separate PR?